HS9_py2/heatsource9/Tests/test_pyheatsource.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""Not implemented with Cython yet. Useful for testing individual functions within vanilla python"""
+
```

Comment:
This Pytest file doesn't work with the Cython version yet, but is useful for testing a default Python-only version. I added it to aid in migration to a hot-swappable calculation class (and Python3)